app/app.js

```diff
@@ -121,6 +121,10 @@ window.run = function() {
             HeaderConfig.cartHeaderContent.imageUrl,
             HeaderConfig.cartHeaderContent.id
         ).then(function(counterBadgeController) {
+            Astro.registerRpcMethod(AstroRpc.names.updateCartCounter, ['count'], function(res, count) {
```

Comment:
Counter badge controller is an astro plugin. We have not had astro register rpc methods yet. I think it makes sense but we might need to talk about whether we want a class like `AstroRpc` in astro as well as one in the scaffold app itself.